README.md

```diff
@@ -372,85 +386,69 @@ _Note: Deployments and changes have a many to one relationship.  Table only cont
   </tr>
 </table>
 
-
+Where the key icon indicates that the ID is generated by the original system, such as GitHub.
 
 # Dashboard 
 
 The dashboard displays all four metrics with daily systems data, as well as a current snapshot of the last 90 days.  
 
-To understand the metrics and intent of the dashboard, please see the [2019 State of DevOps Report.](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf) 
+To understand the metrics and intent of the dashboard, see the [2019 State of DevOps Report](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf). 
 
 
-## Metrics Definitions
+## Metrics definitions
 
 **Deployment Frequency**
 
-
-
 *   The number of deployments per time period: daily, weekly, monthly, yearly. 
 
 **Lead Time for Changes**
 
-
-
-*   The median amount of time for a commit to be deployed into production
+*   The median amount of time for a commit to be deployed into production.
```

Comment:
What's the change here?

Reply:
no change, possibly removing empty lines above.